### PR TITLE
Port PR #1098

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Port PR #1098 into FAQ section
| Fixed ticket? |

### CODE REVIEW

File: faq/theme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+**Q:** Why new-theme directory does not have a package.json?
+
+**A:** If you use PrestaShop's production ZIP release instead of the sources, new-theme directory will not have a package.json. `npm run build` will then fail.
+
+```
+Module build failed: Error: Couldn't find preset "env" relative to directory
+```
+
+You can solve this by changing to a release for development or by copy the necessary json file to your new-theme folder, navigate there and run `npm install`.
```
Comment (PierreRambaud) [inline faq/theme.md:15]: IMHO, the production build shouldn't be tweaked. But the source must be used instead

Comment (matks) [reply faq/theme.md:15]: You're right, the best scenario is to use sources. But I discussed with multiple agency developers who do not 😢  : they download production-ready ZIP and copy the files they need from github.

So what do we do: we explain the best usecase that maybe most people dont follow, or we add a FAQ item for that we know is a real life situation 🗃️

Comment (PierreRambaud) [reply faq/theme.md:15]:
```suggestion
PLEASE DON'T CHANGE THE PRODUCTION ZIP, USE SOURCE AND CREATE A RELEASE IF NEEDED!
```

Comment (matks) [reply faq/theme.md:15]: @PierreRambaud I think I can guess what you mean 😄

**Pull request closed without merging**